spec/dishes_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'customer'
+require 'dishes'
+
+describe Dishes do
+
+  it 'should display the dishes and prices to the customer' do
+    customer = Customer.new
+    expect(customer.check).to eq(@dishes)
```

Comment:
I think `@dishes` has no value here, so it would be `nil`. Please could you test for what `customer.check` should return?

Reply:
Thank you josh, I noticed that now I've fixed it. I used p to get visibility and then I checked the customer.check in irb and referenced the correct @dishes variable instead of referencing a new one in the code 👍